calc/README.md

```diff
@@ -0,0 +1 @@
+# calc
```

Comment:
I would suggest that you either delete this file or replace it with something more useful.